packages/git/node-tests/writer-test.js

```diff
@@ -311,7 +311,7 @@ describe('git/writer', function() {
 
     it('requires id in body', async function() {
       try {
-        await writers.update('master', env.session, 'articles', '1', {
+        await writers.update(env.session, 'articles', '1', {
```

Comment:
@robustdj same for `writers.update()` we need to remove any branch parameters being passed into this function. we might have some of these within the cards of portfolio.